extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/dialect/QuarkusPostgreSQL95Dialect.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.hibernate.orm.deployment.dialect;
+
+import java.sql.DatabaseMetaData;
+import java.sql.SQLException;
+
+import org.hibernate.dialect.PostgreSQL95Dialect;
+import org.hibernate.engine.jdbc.env.spi.IdentifierCaseStrategy;
+import org.hibernate.engine.jdbc.env.spi.IdentifierHelper;
+import org.hibernate.engine.jdbc.env.spi.IdentifierHelperBuilder;
+
+/**
+ * Subclass of PostgreSQL95Dialect fixing schema updates by considering unquoted identifiers as lower case.
+ * This is PostgreSQL's behavior.
+ * See https://github.com/quarkusio/quarkus/issues/1886
+ *
+ * @author Stephane Epardaud
+ * @author Emmanuel Bernard emmanuel@hibernate.org
```

Comment:
Same here.

Reply:
Yeah, please, I don't want my name anywhere near all the bugs I write…